scripts.js

```diff
@@ -0,0 +1,53 @@
+let localStorageBooks = [];
+const div = document.querySelector('#bookListContainer');
+const ul = document.querySelector('#ulBookList');
+const addForm = document.getElementById('addBookForm');
+
+function addBook(cTitle, cAuthor) {
+  const newBook = {
+    title: cTitle,
+    author: cAuthor,
+  };
+  localStorageBooks.push(newBook);
+  localStorage.setItem('localStorageBooks', JSON.stringify(localStorageBooks));
+}
+
+function removeBook(index) {
+  localStorageBooks.splice(index, 1);
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you make use array methods that doesn't mutate the array, e.g. `concat`, `filter`, `slice` instead of  `splice`.